theodinproject/mastermind/computer_logic.rb

```diff
@@ -6,9 +6,9 @@ def find_remaining_combinations(computer_guess, clues)
       @all_combinations = computer_guess.permutation(4).to_a
     end
     if red_pegs > 0
-      possible_matching_positions = [0, 1, 2, 3].combination(red_pegs).to_a
+      possible_correct_positions = [0, 1, 2, 3].combination(red_pegs).to_a # Ex: if red_pegs=2, then possible correct positions could be 0,1 or 0,2 or 1,3 etc
```

Comment:
This method `find_remaining_combinations` has a lot of code which makes it hard to read and understand.

We can extract code inside each if statement to separate method. For e.g.

Code inside `if red_pegs > 0` could be inside a new method.